src/cli/commands/force/lightning/lwc/preview.ts

```diff
@@ -175,8 +175,7 @@ export class Preview extends Setup {
             );
 
             const hasConfigFile =
-                this.configFilePath.length > 0 &&
-                fs.existsSync(this.configFilePath);
+                configFileName.length > 0 && fs.existsSync(this.configFilePath);
```

Comment:
This check should have been against `configFileName` not `configFilePath` since `configFilePath` will always have a default value so it's length will always be >0